pkg/eestream/common.go

```diff
@@ -0,0 +1,10 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package eestream
+
+import (
+	"github.com/spacemonkeygo/errors"
```

Comment:
What's the benefit of using the errors package? Should it become a standard for our code base?

Reply:
this errors package does stack traces. i think we should probably talk about this as a team, but my specific usage of this package was out of habit. if i had to vote i'd vote for https://github.com/zeebo/errs

Reply:
what's the difference between `github.com/spacemonkeygo/errors` and `https://github.com/zeebo/errs` ?

Reply:
not much other than age. zeebo and i wrote spacemonkeygo/errors, and then used it for 4 years, and after 4 years zeebo decided to get rid of some cruft and clean it up to match how we really used it and fix some lessons we had learned